### DISCUSSION TITLE
default Variability for CYP2C8 and CYP1A2 t1/2 (liver)

### DISCUSSION THREAD

**Discussion by tobiasK2001** [Open-Systems-Pharmacology/Forum#1564] [Questions & Problems] [answered]
Dear all,

When I create a population based on an individual with expression profile for CYP1A2 and 2C8 there seems to be a vaiabilty in 1/2 liver: SD is 1440 min.
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/25585248/044529e9-5bb4-4e38-a13c-e4a013f821da)
which is larger than the mean in case for 2C8 or in other words the SD would include zero.
For CYP1A2 the mean is bigger than the SD
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/25585248/959d69f5-616f-4bf0-b487-1441c680e711)
But still I am wondering were this SD is coming from. I thought the variability for Enzyme expression would be linked via the Ontogeny factor (if an ontogeny was defined for PK-Sim). For other CYPs the SD is 0, e.g. for CYP3A4
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/25585248/a4f0ec8f-611a-4863-847b-40a1368ed48f)

Is this SD of 1440 min for CYP1A2 and 2C8 correct or a Bug?

Best, Tobias

P.s. I am using the PK-Sim 11 portable version

**Comment by Yuri05** [comment, marked as answer]
>I thought the variability for Enzyme expression would be linked via the Ontogeny factor (if an ontogeny was defined for PK-Sim).

No, variability of enzyme "global" parameters (like Reference concentration and half lives) are defined separately.

>Is this SD of 1440 min for CYP1A2 and 2C8 correct or a Bug?

Correct.

enzyme | Location | mean half-life [h] | SD [h] | reference |  
-- | -- | -- | -- | -- | --
1A2 | Liver | 39 | 24 | Yang2008/ Faber and Fuhr_2004_Time response of cytochrome P450   1A2 activity on cessation of heavy smoking | http://www.ncbi.nlm.nih.gov/pubmed/15289794
2C8 | Liver | 23 | 19 | Yang2008/ Renwick_2000_Differential maintenance of CYP in   cultured precision-cut human liver slices | https://pubmed.ncbi.nlm.nih.gov/10997941/
2E1 | Liver | 50 | 19 | Yang 2008 / Emery 1999 | https://pubmed.ncbi.nlm.nih.gov/10490907/

**Comment by tobiasK2001** [comment]
Thanks! Maybe it would be good to save these references in in the value origin in the next PK-Sim Version.

Best, Tobais
